### DISCUSSION TITLE
How to prevent the title bar from blinking?

### DISCUSSION THREAD

**Discussion by kaelig** [Abdenasser/neohtop#116] [General]
I just installed 1.1.0 and found the [newly introduced](https://github.com/Abdenasser/neohtop/pull/81) incessant blinking of the title bar to be very distracting to me. I'd love to know how to disable it 🙏🏻

![Screenshot 2024-11-14 at 10 56 17 AM](https://github.com/user-attachments/assets/8e31e127-798e-4a8e-8cef-0c0bdab241f5)
![Screenshot 2024-11-14 at 10 56 07 AM](https://github.com/user-attachments/assets/154b16b6-98a9-4ad8-a0b5-4f94078c36a0)

**Comment by Abdenasser** [comment]
@kaelig if you like you can clone the project and and make the changes to [TitleBar.svelte](https://github.com/Abdenasser/neohtop/blob/main/src/lib/components/TitleBar.svelte) then build the app following the Readme guide
